src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -195,7 +195,41 @@
                            (m "attributes")])
                      (:entities @state))}]])
         (style/create-form-label "Define Expression")
-        (style/create-text-field {})]])}])
+        (style/create-text-field {:ref "expressionname" :defaultValue "this"})]
+
+       [:div {:style {:fontSize "106%" :lineHeight 1 :textAlign "center"}}
+        [:div {:style   {:padding         "0.7em 0" :cursor "pointer"
+                         :backgroundColor (:button-blue style/colors)
+                         :color "#fff" :borderRadius 4
+                         :border          (str "1px solid " (:line-gray style/colors))}
+
+               ;; TODO: what should we show in the UI after submitting?
+               ;; TODO: if no expression entered, do not send one in the ajax request
+               ;; TODO: disable/enable submit button based on the state of entity selection & expression value
+               ;; TODO: diable submit button after submitting
+               ;; TODO: better canned response
+               :onClick (fn [{:keys []}]
+                            (utils/ajax-orch
+                              (paths/submit-method-path workspace)
+                              {:method :post
+                               :data (utils/->json-string {:methodConfigurationNamespace (config "namespace")
+                                                           :methodConfigurationName      (config "name")
```

Comment:
Nit: Prefer to not line these up. Easy to adjust in IntelliJ (if you didn't do this by hand).